app/models/post.rb

```diff
@@ -1,7 +1,17 @@
-# rubocop:disable all
-
 class Post < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+  after_initialize :set_defaults
+
+  private
+
+  def update_posts_counter
+    author.increment!(:postsCounter)
+  end
+
+  def set_defaults
+    self.likes_counter ||= 0
+    self.comments_counter ||= 0
+  end
 end
```

Comment:
- [ ] The Post model should have a method for returning the 5 most recent comments.

  ```
  def recent_comments
    comments.order(created_at: :desc).limit(5)
  end
  ```